BridgeSDK/BridgeAPI/SBBAuthManagerInternal.h

```diff
@@ -73,6 +73,15 @@ extern NSString * const kSBBAuthResetAPI;
 @property (nonatomic, strong) id<SBBObjectManagerProtocol> objectManager;
 @property (nonatomic, strong) SBBUserSessionInfo *placeholderSessionInfo;
 
+// Keep track of whether an auth/reauth call to Bridge is currently in progress.
+@property (nonatomic, assign) BOOL authCallInProgress;
+
+// Keep track of the URL session task for an in-progress auth/reauth call to Bridge.
+@property (nonatomic, assign) NSURLSessionTask *inProgressAuthURLSessionTask;
```

Comment:
Shouldn't this be `weak` rather than `assign`? Sorry, can't remember.

Reply:
Whoops. In fact it should be `strong`

Reply:
or maybe just not exist ;)